rmgpy/constraints.py

```diff
@@ -31,6 +31,32 @@
 
 from rmgpy.species import Species
 
+def pass_cutting_threshold(species):
+    """
+    Pass in either a `Species` or `Molecule` object and checks whether it passes 
+    the speciesCuttingThreshold set by the user. The default value is 20. If yes,
+    returns `True` for passing cutting threshold.
+    """
+
+    from rmgpy.rmg.input import get_input
```

Comment:
Same with this import, though I suspect this one will cause a problem if it is moved.

Reply:
Yeah it did cause an error once I move it up to the front.